scss/main.scss

```diff
@@ -1 +1,93 @@
 @use 'vars';
+
+// Track Circle Class
+@mixin circle($color, $size) {
+  background-color: $color;
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+  display: inline-block;
+}
+// Footer Circle Class
+@mixin footer-circle($color, $size){
+  @include circle($color, $size);
+  border: 1px solid $color;
+  box-sizing: border-box;
+  right: 0%;
+  bottom: 0%;
```

Comment:
why wouldnt the circle mix-in by itself work for the footer circles?

Reply:
I added a sep class because each of the footer circles has a border and just figured it'd be easier to use a class than copy border 3 times?